src/libLookup/Lookup.cpp

```diff
@@ -3658,8 +3658,20 @@ void Lookup::RectifyTxnShardMap(const uint32_t oldNumShards,
       continue;
     }
     for (const auto& tx : shard.second) {
-      unsigned int correct_shard = tx.GetShardIndex(newNumShards);
-      tempTxnShardMap[correct_shard].emplace_back(tx);
+      unsigned int fromShard = tx.GetShardIndex(newNumShards);
```

Comment:
@KaustubhShamshery For the case of the old DS shard (few lines above this), should we skip the transactions there, or do another check to see if the `toShard` and `fromShard` for those don't match?

Reply:
@ansnunez they might have priority set to true. so sending them to new ds is appropriate,